### ISSUE TITLE
Infinite recursion/loop with `indent-region'

### ISSUE THREAD

**Issue by Alexander-Shukaev** [emacs-evil/evil-surround#113]
Let's roll:

```
$ emacs -Q --eval '(add-to-list '"'"'load-path (concat user-emacs-directory "elpa/evil-YYYYMMDD.ABC"))' --eval '(add-to-list '"'"'load-path (concat user-emacs-directory "elpa/evil-surround-YYYYMMDD.ABC"))' --eval '(require '"'"'evil)' --eval '(evil-mode)' --eval '(require '"'"'evil-surround)' --eval '(global-evil-surround-mode)'
```

Move to some word and type `ySiw"`.  Observe infinite hang due to recursion/loop with `indent-region` (see the "Indenting..." message in the minibuffer).  Press `C-g C-g C-g` to break out and observe the correct text modification.

Please, fix.

**Comment by ninrod** [comment]
thank you for the bug report. 

Can't reproduce with GNU Emacs 25.2.1 (x86_64-unknown-linux-gnu, GTK+ Version 3.22.10) of 2017-04-22

What is your version?

**Comment by Alexander-Shukaev** [comment]
Emacs 26 built yesterday from Git.

**Comment by ninrod** [comment]
fixed in GNU Emacs 27.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.26) of 2017-12-02
